exporters/zipkin/build.gradle

```diff
@@ -20,8 +20,7 @@ dependencies {
     implementation "io.zipkin.reporter2:zipkin-sender-okhttp3"
 
     testImplementation project(':sdk:testing')
-    testImplementation "com.google.guava:guava",
```

Comment:
Error prone wanted to use guava types on the left side for static final constants. Instead I removed guava

Reply:
Guava is not a problem in test, so this is not necessary an improvement.